packages/messaging/test/sw-controller.test.ts

```diff
@@ -638,162 +638,164 @@ describe('Firebase Messaging > *SwController', () => {
       expect(event.notification.close.callCount).to.equal(1);
     });
 
-    it('should open click_action URL for FCM notification (third-party so no window client access)', async () => {
-      const clickAction = '/test-click-action';
-      const waitUntilSpy = sandbox.spy();
-      const event: any = {
-        notification: {
-          data: {
-            FCM_MSG: {
+    ['link', 'click_action'].forEach(key => {
```

Comment:
nit: prefer "for (... of ...) {"

go/tsstyle#iterating-arrays

Reply:
Sure, why not. I don't think you'll find any style guide conformance around here :smile: I'll rewrite all of these eventually...